User-Stories/Developer-User-Stories/Developer.md

```diff
@@ -0,0 +1,29 @@
+#Developer
```

Comment:
I am not sure whether the described tasks and role fit a "normal" Software developer daily work. The developers I know care about good code, their main objective is to write as efficient as possible well designed and maintainable code

Reply:
I agree, that these are the main tasks. But assuming an educated OS-developer I would count these tasks in. Don't you think, this should be the result of a successful OSS awareness programm.

Reply:
Sure I agree.  But the objective of our effort here is alos to offload all the stakeholders as much as possible for the work license compliance is causing, so I would not focus on these things in the role description since these tasks shall be done by the toolchian

Reply:
This implies, that all and everything needs to be build. I guess we might agree on the fact that we will address a modern CI/CD-scenario. But even here, we will find some solution components that will not appear in the build lists, such as infrastructure components like DBs, App-servers, etc. Whom we should make responsible for, if not the developers to complete this list?

Reply:
Isn't there a way to grab this data also automatically?
In the end either the developers or the architects are responsible for the completness and correectness